### PR TITLE
Adding support for environment variables endpoint.

### PR DESCRIPTION
**Pull request by paryank**

### CODE REVIEW

File: src/Response/VariableResponse.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    public $name;
+    public $value;
+
+    /**
+     * VariableResponse constructor.
+     * @param object $variable
+     */
+    public function __construct($variable)
+    {
+        $this->name = $variable->name;
```
Comment (typhonius) [inline src/Response/VariableResponse.php]: Can we also pass back `links`. For the other classes I've taken an approach of converting _links to links as code sniffing tools complained about having a class property starting with an underscore so:

```
$this->links = $variable->_links;
```

File: tests/Endpoints/VariablesTest.php [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+namespace AcquiaCloudApi\Tests\Endpoints;
+
+use AcquiaCloudApi\Endpoints\Variables;
+use AcquiaCloudApi\Tests\CloudApiTestCase;
+
+class VariablesTest extends CloudApiTestCase
+{
+
+    public $properties = [
+        'name',
```
Comment (typhonius) [inline tests/Endpoints/VariablesTest.php]: Add links here too to test the above.

File: src/Response/VariableResponse.php (hunk 1 above)
Comment (paryank) [reply src/Response/VariableResponse.php]: Done. Thanks

**Comment by typhonius** [comment]
Thank you! Merged in 1b1e9e45f4f2b735f98fa43ad06ec6bdcaf86a3b

**Pull request closed without merging**